docs/react-jss.md

```diff
@@ -153,6 +153,66 @@ const App = () => <Button fontStyle="italic">Submit</Button>
 
 The above code will compile to
 
+```html
+<div id="root">
+  <button class="myButton-1-25">
+    <span class="myLabel-1-26">
+      Submit
+    </span>
+  </button>
+</div>
+```
+
+and
+
+```css
+.myButton-1-25 {
+  padding: 10px;
+}
+.myLabel-1-26 {
+  display: block;
+  color: red;
+  font-weight: bold;
+  font-style: italic;
+}
+```
+## Adding Component Prefix Option
```

Comment:
1. I would probably write them lower case
2. headlines need to be added to the TOC above manually